### PR TITLE
VACMS-4258: update entity_reference_validators

### PR DESCRIPTION
**Pull request by acrollet**
## Description

See #4258 

## Testing done

Added behavioral test to confirm module functionality.

## QA steps

Asa  user with the content admin role
- [ ] Edit a R&S node and ensure that a form validation error is shown when attempting to save with multiple identical benefit hubs referenced

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [x] `Core Application Team`
- [ ] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [x] No announcement is needed for this code change. 
  - [x] Merge & carry on unburdened by announcements

### CODE REVIEW

File: tests/behavioral/cypress/support/commands.js [hunk 1]
```diff
@@ -69,8 +69,16 @@ Cypress.Commands.add('drupalAddUserWithRole', (role, username, password) => {
...
-  return new Cypress.Promise(resolve => {
-    resolve($iframe.contents().find('body'));
+Cypress.Commands.add('iframe', { prevSubject: 'element' }, ($iframe, callback = () => { }) => {
+  return cy
+    .wrap($iframe)
+    .should(iframe => expect(iframe.contents().find('body')).to.exist)
+    .then(iframe => cy.wrap(iframe.contents().find('body')))
+    .within({}, callback)
+});
+
```
Comment (ndouglas) [inline tests/behavioral/cypress/support/commands.js:79]: I figure this involved some suffering.

File: tests/behavioral/cypress/support/commands.js [hunk 2]
```diff
@@ -69,8 +69,16 @@ Cypress.Commands.add('drupalAddUserWithRole', (role, username, password) => {
...
+Cypress.Commands.add("type_ckeditor", (element, content) => {
+  cy.window().then((win) => {
+    win.CKEDITOR.instances[element].setData(content);
```
Comment (ndouglas) [inline tests/behavioral/cypress/support/commands.js:82]: This too, I bet 😭

File: tests/behavioral/cypress/integration/common/i_wait_seconds.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import { Given } from "cypress-cucumber-preprocessor/steps";
+
+Given(`I wait {string} seconds`, (seconds) => {
+  cy.wait(seconds * 1000);
+});
```
Comment (ndouglas) [inline tests/behavioral/cypress/integration/common/i_wait_seconds.js:5]: ❤️

**Comment by acrollet** [comment]
New test passed before the module update:
Screen Shot 2021-02-23 at 12 45 02 PM

**Review by ndouglas — APPROVED** [review]
unnamed (2)

Screen Shot 2021-02-23 at 2 56 58 PM

File: tests/behavioral/cypress/support/commands.js (hunk 1 above)
Comment (acrollet) [reply tests/behavioral/cypress/support/commands.js:79]: you figure correctly.

**Comment by acrollet** [comment]
Tests did not report but all passed:

Screen Shot 2021-02-23 at 1 32 59 PM

**Pull request merged**